diff -urN kdepimlibs-4.8.2.orig/akonadi/contact/kcm/akonadicontact_actions.desktop kdepimlibs-4.8.2/akonadi/contact/kcm/akonadicontact_actions.desktop
--- kdepimlibs-4.8.2.orig/akonadi/contact/kcm/akonadicontact_actions.desktop	2012-01-18 22:07:05.000000000 +0400
+++ kdepimlibs-4.8.2/akonadi/contact/kcm/akonadicontact_actions.desktop	2012-04-16 02:00:06.548204967 +0400
@@ -118,6 +118,7 @@
 X-KDE-Keywords[pl]=kaddressbook, konfiguruj, ustawienia, kontakt, działania
 X-KDE-Keywords[pt]=kaddressbook, configurar, configuração, contacto, acções
 X-KDE-Keywords[pt_BR]=kaddressbook,configurar,configurações,contato,ações
+X-KDE-Keywords[ru]=kaddressbook, configure, settings, contact, actions, адресная книга, настроить, настройки, контакт, действия
 X-KDE-Keywords[sr]=kaddressbook,configure,settings,contact,actions,К‑адресар,подесити,подешавање,поставке,контакт,радње
 X-KDE-Keywords[sr@ijekavian]=kaddressbook,configure,settings,contact,actions,К‑адресар,подесити,подешавање,поставке,контакт,радње
 X-KDE-Keywords[sr@ijekavianlatin]=kaddressbook,configure,settings,contact,actions,K‑adresar,podesiti,podešavanje,postavke,kontakt,radnje
diff -urN kdepimlibs-4.8.2.orig/kresources/kresources.desktop kdepimlibs-4.8.2/kresources/kresources.desktop
--- kdepimlibs-4.8.2.orig/kresources/kresources.desktop	2012-01-18 22:07:05.000000000 +0400
+++ kdepimlibs-4.8.2/kresources/kresources.desktop	2012-04-16 02:00:37.470205642 +0400
@@ -130,6 +130,7 @@
 X-KDE-Keywords[pl]=zasoby,zasoby konnector,zasoby kontaktu,zasoby kalendarz,zasoby notatek,zasoby alarmu,imap
 X-KDE-Keywords[pt]=recursos,recurso do konector,recurso de contacto,recurso de calendário,recurso de notas,recurso de alarme,IMAP
 X-KDE-Keywords[pt_BR]=recursos,recurso do konnector,recurso do contato,recurso do calendário, recurso de notas, recurso do alarme,imap
+X-KDE-Keywords[ru]=resources,konnector resource,contact resource,calendar resource,notes resource,alarm resource,imap,ресурсы,источники,соединитель,контакты,календарь,календари,заметка,заметки,сигналы,напоминание
 X-KDE-Keywords[sr]=resources,konnector resource,contact resource,calendar resource,notes resource,alarm resource,imap,ресурси,ресурс конектора,ресурс контаката,ресурс календара,ресурс белешки,ресурс аларма,ИМАП
 X-KDE-Keywords[sr@ijekavian]=resources,konnector resource,contact resource,calendar resource,notes resource,alarm resource,imap,ресурси,ресурс конектора,ресурс контаката,ресурс календара,ресурс белешки,ресурс аларма,ИМАП
 X-KDE-Keywords[sr@ijekavianlatin]=resources,konnector resource,contact resource,calendar resource,notes resource,alarm resource,imap,resursi,resurs konektora,resurs kontakata,resurs kalendara,resurs beleški,resurs alarma,IMAP
